### PR TITLE
Adds speakers, sponsors etc

### PR DESCRIPTION
**Pull request by MarijeP**
* First 7 speakers
* Adds AWS, Estalea
* Update the footer to say 2019 (cough https://twitter.com/adrianmoisey/status/1087297706857512961)

### CODE REVIEW

File: _data/speakers.yml [hunk 1]
```diff
@@ -14,6 +14,139 @@
...
+      Simon started out life as an astronomer, used to be an engineer, and is now trying to avoid managerial tasks. Currently in charge of scientific computing for the MeerKAT radio telescope (<a href='http://www.ska.ac.za'>ska.ac.za</a>), he spends a significant amount of his coding time trying to avoid writing C, and the rest of the time trying to use <code>sys._getframe()</code> in ways that annoy his colleagues.
+    title: >
+       A Sky Full of Stars
+    abstract: >
+      Hundreds of billions of galaxies each with hundreds of billions of stars; how do you create a machine to help make sense of it all? This is the story of building MeerKAT - the most powerful radio telescope in the world. This is a follow on Simon's <a href='https://scaleconf.org/speakers/2013'>ScaleConf 2013 talk</a> which set out the plans for delivering MeerKAT, and how they planned to address the scale challenges.
+    video:
+
+helge_reikeras:
+    name: Helge Reikeras
+    company: Offerzen
```
Comment (adrianmoisey) [inline _data/speakers.yml]: `OfferZen`

File: _layouts/base.html [hunk 2]
```diff
@@ -85,7 +85,7 @@ <h3>Community</h2>
     </div>
 
     <div style='position:relative; margin-top:-2px; padding:50px 0 0 0; background:#fff;'>
-      <p style='background:#5bc8df; color:#fff; text-align:center; padding:10px; margin:0;'>&copy; ScaleConf 2018</p>
+      <p style='background:#5bc8df; color:#fff; text-align:center; padding:10px; margin:0;'>&copy; ScaleConf 2019</p>
```
Comment (adrianmoisey) [inline _layouts/base.html]: use `{{ 'now' | date: "%Y" }}` instead of 2019 and you'll never need to update it, ever again.

**Comment by MarijeP** [comment]
THANKS ADRIAN.

**Comment by adrianmoisey** [comment]
NO WORRIES

**Pull request merged**